readme.md

```diff
@@ -950,12 +950,15 @@ You can use the [`tough-cookie`](https://github.com/salesforce/tough-cookie) pac
 
 ```js
 const got = require('got');
-const {CookieJar} = require('tough-cookie');
-
-const cookieJar = new CookieJar();
-cookieJar.setCookie('foo=bar', 'https://www.google.com');
-
-got('https://google.com', {cookieJar});
+const { CookieJar } = require('tough-cookie');
+const util = require('util');
+
+const yourFunction = async () => {
```

Comment:
You don't need to name the function. Just execute it immediately:

```js
(async () => {
    ...
})();
```

Reply:
that throws SyntaxError: await is only valid in async function

Reply:
ok